poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
@allensallinger it looks like you are using an old version of poetry

Reply:
Upgraded the poetry version and I can now update the lock file.